src/In.ProjectEKA.HipService/Logger/Log.cs

```diff
@@ -0,0 +1,44 @@
+
+namespace In.ProjectEKA.HipService.Logger
+{
+    using System;
+    using Serilog;
+    
+    public class Log
+    {
+        public static ILogger Logger
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Provide a getter for 'Logger' or replace the property with a 'SetLogger' method.](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=4954915)